app/code/community/Bolt/Boltpay/Helper/Data.php

```diff
@@ -147,8 +147,6 @@ public function buildOnCheckCallback($checkoutType, $isVirtualQuote = false)
      */
     public function buildOnSuccessCallback($successCustom, $checkoutType)
     {
-        $saveOrderUrl = $this->getMagentoUrl("boltpay/order/save/checkoutType/$checkoutType");
```

Comment:
huh what was this used for, weird

Reply:
@matt-thomason 1.x version of the plugin used that